morebits.js

```diff
@@ -2702,11 +2702,10 @@ Morebits.wiki.page = function(pageName, currentAction) {
 		ctx.onMoveSuccess = onSuccess;
 		ctx.onMoveFailure = onFailure || emptyFunction;
 
-		if (!ctx.editSummary) {
-			ctx.statusElement.error('Internal error: move reason not set before move (use setEditSummary function)!');
-			ctx.onMoveFailure(this);
-			return;
+		if (!fnPreflightChecks('move', ctx.onMoveFailure)) {
```

Comment:
```suggestion
		if (!fnPreflightChecks.call(this, 'move', ctx.onMoveFailure)) {
```
because fnPreflightChecks uses `this`.

Following 911, this would need to be either:
```
var error = fnPreflightChecks.call(this, 'move', ctx.onMoveFailure);
if (error) {
    return error;
}
```
with fnPreflightChecks returning a rejected promise (in case of error) or null (in case of no error)

Or alternatively 
```
var error = fnPreflightChecks.call(this, 'move');
if (error) {
    return returnError.call(this, error.code, error.text);
}
```
with fnPreflightChecks returning an object containing the error code and text.

Reply:
Jah.  I'm about to be AFK for 2 weeks (and as stated would come after #911) but wanted to get it up for "yea or nay"